docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,673 @@
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Place the example code in `main()` after `robot = await connect()`.
+
+```python
+from viam.components.movement_sensor import MovementSensor
+
+robot = await connect() # Refer to CODE SAMPLE tab code
+my_imu = MovementSensor.from_robot(robot, "my-imu")
+
+# Get the current linear acceleration
+lin_accel = await my_imu.get_linear_acceleration()
+print(f"my-imu linear acceleration in the x direction: {lin_accel.x}")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+Place the example code in the `main()` function after `robot, err := client.New(...)`.
+
+```go
+import (
+  "context"
+  "time"
+
+  "github.com/edaniels/golog"
+
+  "go.viam.com/rdk/components/movementsensor"
+)
+
+robot, err := client.New() // Refer to CODE SAMPLE tab code
+// Grab the movement sensor from the robot
+myIMU, err := movementsensor.FromRobot(robot, "my-imu")
+if err!=nil { 
+  fmt.Println(err) 
+}
+
+// Get the current linear acceleration
+linAccel, err := myIMU.LinearAcceleration(context.Background(), map[string]interface{}{})
+if err!=nil { 
+  fmt.Println(err) 
+}
+fmt.Println("my-imu LinearAcceleration in the x direction: ", linAccel.X)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+Different movement sensors provide different data, so be aware that not all of the methods below are supported by all movement sensors.
+You can run `GetProperties` on your sensor for a list of its supported methods.
+
+Method Name | Description | Models That Support This Method
+----------- | ----------- | -------------------------------
+[GetPosition](#getposition) | Gets the current latitude, longitude and altitude. | GPS models
+[GetLinearVelocity](#getlinearvelocity) | Gets the current linear velocity as a 3D vector. | GPS models
+[GetAngularVelocity](#getangularvelocity) | Gets the current angular velocity as a 3D vector. | IMU models and `gyro-mpu6050`
+[GetLinearAcceleration](#getlinearacceleration) | Gets the current linear acceleration as a 3D vector. | IMU models,  `accel-adxl345`, and `gyro-mpu6050`
+[GetCompassHeading](#getcompassheading) | Gets the current compass heading in degrees. | GPS models and `imu-vectornav`
+[GetOrientation](#getorientation) | Get the current orientation. | IMU models
+[GetProperties](#getproperties) | Get the supported properties of this sensor. | all models
+[GetAccuracy](#getaccuracy) | Get the accuracy of the various sensors. | GPS models
+[GetReadings](#getreadings) | Obtain the measurements/data specific to this sensor. | all models
+[DoCommand](#docommand) | Sends or receives model-specific commands. | all models
+
+In addition to the information below, see the [Go SDK docs](https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#MovementSensor)
+or [Python SDK docs](https://python.viam.dev/autoapi/viam/components/movement_sensor/index.html#).
+
+### GetPosition
+
+Report the current GeoPoint (latitude, longitude) and altitude (in millimeters).
+

```

Comment:
I think adding this info here as well would be helpful
```suggestion

Only supported by GPS models.

```

Reply:
Do you think it should be added to every method or just this one?

Reply:
All the ones that aren't supported by all models

Reply:
done